examples/transpose/transpose_block_numa.cpp

```diff
@@ -583,8 +583,8 @@ int main(int argc, char* argv[])
         "hpx.run_hpx_main!=1",
         "hpx.numa_sensitive=2",  // no-cross NUMA stealing
         // block all cores of requested number of NUMA-domains
-        hpx::util::format("hpx.cores={}", numa_nodes * num_cores),
-        hpx::util::format("hpx.os_threads={}", numa_nodes * pus.second)
+//        hpx::util::format("hpx.cores={}", numa_nodes * num_cores),
+//        hpx::util::format("hpx.os_threads={}", numa_nodes * pus.second)
```

Comment:
Can we remove the commented code sections here? Or is this an oversight?

Reply:
The old example was crashing on startup because it set those params and hey didn't work as expected with the resource partitioner stuff. I commented out stuff to make the example run without error, but it doesn't not really work any more and it needds further work. For now, I think this should be left 'as is' until I get time to patch it up and make it work with the new design of allocators and RP.